app/uk/gov/hmrc/transitmovements/models/mongo/read/MongoMovementEori.scala

```diff
@@ -29,21 +29,24 @@ object MongoMovementEori {
     BsonDocument(
       "_id"                  -> 1,
       "enrollmentEORINumber" -> 1,
-      "clientId"             -> 1
+      "clientId"             -> 1,
+      "isTransitional"       -> 1
     )
 }
 
 case class MongoMovementEori(
   _id: MovementId,
   enrollmentEORINumber: EORINumber,
-  clientId: Option[ClientId]
+  clientId: Option[ClientId],
+  isTransitional: Option[Boolean]
 ) {
 
   @transient lazy val movementWithEori: MovementWithEori =
     MovementWithEori(
       _id,
       enrollmentEORINumber,
-      clientId
+      clientId,
+      isTransitional.getOrElse(true)
```

Comment:
I assume you have tested this when we don't have this field in database and it's returning ```None```.  Not sure if it can return ```null``` and then when we use ```isTransitional.getOrElse``` will ends up with ```NullPointerException```